_overviews/cheatsheets/index.md

```diff
@@ -397,6 +397,232 @@ Some(3) match {
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(_) =&gt; false
+  case None    =&gt; true
+}</code></pre></td>
+      <td>True if empty</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.nonEmpty</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(_) =&gt; true
+  case None    =&gt; false
+}</code></pre></td>
+      <td>True if not empty</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.size</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+case Some(_) =&gt; 1
+case None    =&gt; 0
+
+      </code></pre></td>
+      <td>Zero if empty, otherwise one</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.orElse(Some(y))</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(x) =&gt; Some(x)
+  case None    =&gt; Some(y)
+}</code></pre></td>
+      <td>Evaluate and return alternate optional value if empty</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.getOrElse(y)</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(x) =&gt; x
+  case None    =&gt; y
+}</code></pre></td>
+      <td>Evaluate and return default value if empty</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.get</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(x) =&gt; x
+  case None    =&gt; throw new Exception
+}</code></pre></td>
+      <td>Return value, throw exception if empty</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.orNull</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(x) =&gt; x
+  case None    =&gt; null
+}</code></pre></td>
+      <td>Return value, null if empty</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.filter(f)</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(x) if f(x) =&gt; Some(x)
+  case _               =&gt; None
+}</code></pre></td>
+      <td>Optional value satisfies predicate</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.filterNot(f(_))</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(x) if !f(x) =&gt; Some(x)
+  case _                =&gt; None
+}</code></pre></td>
+      <td>Optional value doesn't satisfy predicate</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.exists(f(_))</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(x) if (f(x)) =&gt; true
```

Comment:
maybe better drop unnesesary brakets `(f(x))`  to `f(x)` ?

Reply:
Agree.  I've removed them.